spec/factories/firm.rb

```diff
@@ -25,11 +25,19 @@
     longitude { Faker::Address.longitude.to_f.round(6) }
     status :independent
 
-    factory :onboarded_firm, traits: [:with_advisers, :with_offices], aliases: [:publishable_firm] do
-      advisers_count 1
+    transient do
       offices_count 1
     end
 
+    after(:create) do |firm, evaluator|
+      create_list(:office, evaluator.offices_count, firm: firm)
+      firm.reload
+    end
+
+    factory :onboarded_firm, traits: [:with_advisers] do
```

Comment:
Agreed. We need to fix `Firm#publishable?` then they really will be the same. However at this point there is a difference. Will probably propose a change to do this later. This at least is the minimum change needed to get rad_consumer upgraded to use the same version of this library as rad is using.